src/plugins/vis_types/heatmap/public/vis_type/heatmap.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { Position } from '@elastic/charts';
+
+import { AggGroupNames } from '../../../../data/public';
+import { ColorSchemas } from '../../../../charts/public';
+import { VIS_EVENT_TO_TRIGGER, VisTypeDefinition } from '../../../../visualizations/public';
+import { HeatmapTypeProps, HeatmapVisParams, AxisType, ScaleType } from '../types';
+import { toExpressionAst } from '../to_ast';
+import { getHeatmapOptions } from '../editor/components';
+import { SplitTooltip } from './split_tooltip';
+
+export const getHeatmapVisTypeDefinition = ({
+  showElasticChartsOptions = false,
+  palettes,
+  trackUiMetric,
+}: HeatmapTypeProps): VisTypeDefinition<HeatmapVisParams> => ({
+  name: 'heatmap',
+  title: i18n.translate('visTypeHeatmap.heatmap.heatmapTitle', { defaultMessage: 'Heat map' }),
+  icon: 'heatmap',
+  description: i18n.translate('visTypeHeatmap.heatmap.heatmapDescription', {
+    defaultMessage: 'Shade data in cells in a matrix.',
+  }),
+  toExpressionAst,
+  getSupportedTriggers: () => [VIS_EVENT_TO_TRIGGER.filter],
+  visConfig: {
+    defaults: {
+      type: 'heatmap',
+      addTooltip: true,
+      addLegend: true,
+      enableHover: false,
+      legendPosition: Position.Right,
+      colorsNumber: 4,
+      colorSchema: ColorSchemas.Greens,
```

Comment:
Here the heatmap visualizations default to using the "Greens" (aka positive) color palette. However, in Lens, the heatmap visualization defaults to using the temperature color palette. Currently it doesn't look like Visualize offers the option for a temperature color palette. If we are unable or unwilling to add the temperature color palette and apply it as the default for consistency, should we change default palette to be "Green to Red" (aka status) to make it a little closer conceptually?

Reply:
@MichaelMarcialis we are going to use the EUI palettes on the heatmap on a next iteration and then we will add the temperature palette as the default https://github.com/elastic/kibana/issues/116917
For now, I will follow your suggestions and all the new heatmap visualizations will default to the GreenToRed color schema.